src/components/results-list/ResultsDataFetcher.tsx

```diff
@@ -0,0 +1,136 @@
+import { FC, useEffect, useState, useContext } from "react";
+import { FhirContext } from "../fhir/FhirContext";
+
+import LoadingSpinner from "../UI/LoadingSpinner";
+import ModalWrapper from "../UI/ModalWrapper";
+import { ModalState } from "../UI/ModalWrapper";
+import ResultsList from "../results-list/ResultsList";
+
+import { Patient, Observation } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/models-r4";
+
+const FHIR_URL = process.env.REACT_APP_FHIR_URL;
+
+type trimmedObservation = { cDnaChange: string | undefined; observationId: string | undefined };
+type patientResult = {
+  firstName: string | undefined;
+  lastName: string | undefined;
+  patientId: string | undefined;
+  observations: trimmedObservation[];
+};
+export type parsedResultsModel = patientResult[];
```

Comment:
I think the convention for types is to have a capital first letter, or am I getting my languages mixed up?